sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinModelsRestClient.cs

```diff
@@ -152,17 +152,25 @@ internal Response Update(string id, IEnumerable<string> modelUpdates, Cancellati
 
         // The strings are already json, so we do not want them to be serialized.
         // Instead, the payloads need to be concatenated into a json array.
-        private HttpMessage CreateAddRequest(IEnumerable<string> models)
+        private HttpMessage CreateAddRequest(IEnumerable<string> models, DigitalTwinModelsAddOptions digitalTwinModelsAddOptions)
         {
-            HttpMessage message = _pipeline.CreateMessage();
-            Request request = message.Request;
+            var message = _pipeline.CreateMessage();
```

Comment:
nit: I would keep the explicit type, not clear from the method invocation.

Reply:
This code is copied directly from the protocol layer. I'd rather not change it because of that

Reply:
Agreed. I don't always love the generated code, but we're better off keeping it the way it is, even when we copy/modify into the customized code. It makes it easier for regen.

All I ask is we add comments where we make changes so it is clear to future regenerators.